make.sh

```diff
@@ -110,7 +110,7 @@ trap 'rm -f "$pub" "$sig" "$zip"' EXIT
 # zip up the crx dir
 cwd=$(pwd -P)
 (cd "$dir" && ../../utils/create_zip.py -n "$cwd/$zip" -x "../../.build_exclusions" .)
-echo >&2 "CWS crx package has sha1sum: `sha1sum "$cwd/$zip"`"
+echo >&2 "CWS crx package has sha256sum: `openssl dgst -sha256 -r "$cwd/$zip"`"
```

Comment:
There's nothing obvious here that says that this unusual `openssl dgst -sha256 -r` syntax is needed for OS X support. As an alternative, you could add something to the top like this:

```
if platform is OS X:
	pkg_hash_cmd=openssl dgst -sha256 -r
else:
	pkg_hash_cmd=sha256sum
```

and then use `$pkg_hash_cmd` as needed.